README.md

```diff
@@ -72,11 +72,18 @@ These ambitious startups seem very risky when they're just starting up. They nee
 We make angel investments in great founders who have found an important problem to solve. Then, we help set an ambitious vision and successfully create it.
 
 ## Our Investment Thesis
-First, we look for a startup that's found a group of Users with a important, painful, and valuable Problem. We call this having an "**Important Mission**."
 
-Next, we look for founders who are trying to create an ambitious solution that's first, best, or unique. We call this having a "**Bold Vision**."
-
-Finally, we look for founders with the passion, guts, and ability to actually create it. We call this having the "**Right Team**".
+- **Type**: Software companies
+- **Geography**: Tier-1 Canadian Cities, especially Toronto
```

Comment:
I wouldn't necessarily restrict this to "Tier-1" Canadian Cities since the definition is a little narrow/unclear. I'd rather emphasize: "Fully based out of Canada, preferably Toronto"